request_handlers.cc

```diff
@@ -20,7 +27,9 @@ void EchoRequestHandler::handleRequest(const char* request_buffer,
   response_buffer_size = header_size + request_buffer_size;
 
   // TODO: need to check if we need to resize response buffer, if response is huge
-  response_buffer = new char[response_buffer_size];
+  // Zero out memory using memset
+  std::memset((void *)&response_buffer, 0, sizeof(response_buffer));
+  response_buffer = new char[response_buffer_size + 1];
```

Comment:
Can we `memset` to 0 this portion of memory (as well as any other `new` calls below)?